Directory.Build.targets

```diff
@@ -10,8 +9,7 @@
      suppress all targets for TestProjects using ExcludeFromBuild. -->
     <ExcludeFromBuild Condition="'$(IsPackable)' != 'true' and
         '$(SkipTestBuild)' == 'true' and
-        ('$(IsTestProject)' == 'true' or
-         '$(IsUnitTestProject)' == 'true' or
```

Comment:
Arcade SDK sets `$(IsTestProject)` based on `$(IsUnitTestProject)`; do not need to check both after Arcade (Tests.props) import